tests/config/dapr_postgres_state.yaml

```diff
@@ -20,10 +20,10 @@ spec:
   version: v1
   initTimeout: 1m
   metadata:
-  - name: connectionString
-    value: "host=dapr-postgres-postgresql.dapr-tests.svc.cluster.local user=postgres password=example port=5432 connect_timeout=10 database=dapr_test"
-  - name: table
-    value: querytable
+    - name: connectionString
+      value: "host=dapr-postgres-postgresql.dapr-tests.svc.cluster.local user=postgres password=example port=5432 connect_timeout=10 database=dapr_test"
+    - name: table
+      value: statetable
```

Comment:
Why this change is needed?

Reply:
It doesn't matter all that much, but the table name is conflicting with the [query state store](https://github.com/dapr/dapr/blob/e57a91fc8d2930eca71c8fdbf20084ed18161fc2/tests/config/dapr_postgres_query_state.yaml#L26).